src/Configuration/Config.Azure.KeyVault.Secrets/samples/settings.json

```diff
@@ -0,0 +1,5 @@
+﻿{
+  "CertificateThumbprint": "",
+  "Vault": "",
```

Comment:
@BrennanConroy what did the previous provider use for the config fields? We *can* make changes here if they're valuable but it will force users to do more than just updating a dependency (they may have to change how they are configuring production services, which should be called out).

Reply:
This is a sample, so seems perfectly fine to change the name